Support/Multitenancy/Multiple-Ironic-conductors/README.md

```diff
@@ -0,0 +1,79 @@
+# Metal3 Scalability with Fake Nodes and Multiple Ironics
+
+This directory stores the code and configuration for the __Metal3 Scalability with
+Fake Nodes and Multiple Ironics__. Its purpose is to have a way of testing
+[Metal3](https://metal3.io) provisioning on multiple nodes, without needing
+too much hardware resource.
+
+## Requirements
+
+To perform this experiment, you will need:
+
+- A fresh Ubuntu machine with passwordless sudo.
+- `docker` is installed and can be ran without sudo.
+- Some tools, of which most/all could be installed with the script `vm-setup.sh`.
+
+## Quick Start
+
+For a quick start, here are the steps you need:
+
+- Install some additional tools, add libvirt network
+(named `baremetal`) by using `./vm-setup.sh`.
+- Edit the [config.sh](config.sh) file with suitable configuration. We recommend
+starting with small amount of nodes and clusters, just so you know the
+setup works. Provisioning large numbers of nodes (500-1000) was tested
+and worked well, but that will require large machine and longer time to proceed.
+
+For easy comparison, here is the spec of the machine we used to provision 1000
+single-node clusters:
+
+- CPUs: 20c
+- RAM: 64Gb
+- Hard disk: 750Gb
+
+The whole process took around 48h to finish.
+
+- Run the whole process with `./Init-environment.sh`
+
+## More detailed explanation
+
+In this part, we will explain the purpose of steps taken in `Init-environment.sh`:
+
+- `start_image_server.sh`: starts a nginx server running locally on port 8080. This
+server will be used to store the images that ironic uses for provisioning the nodes.
```

Comment:
```suggestion
   server will be used to store the images that ironic uses for provisioning the nodes.
```
When you need to line split a long list item, please indent by 3 so its more readable in markdown. This occurs many times here, mentioning only once.